scripts/sizeSnapshot/webpack.config.js

```diff
@@ -71,14 +71,14 @@ async function getSizeLimitBundles() {
     },
     {
       // vs https://bundlephobia.com/result?p=react-popper
-      name: '@material-ui/core/Popper',
+      name: 'Popper',
```

Comment:
diffs are tracked via `name`. Should've named this `id` instead to make its purpose clear. 

We can introduce a human `label` in another PR.

Reply:
We shouldn't have any collision (or few exceptions) between the modules we are exporting. How is this an issue?